### DISCUSSION TITLE
rtx 3080ti low maxpower while dual mining eth + ton

### DISCUSSION THREAD

**Discussion by orhangkk** [Lolliedieb/lolMiner-releases#1249] [Q&A]
![issue windows](https://user-images.githubusercontent.com/99587236/153748793-2302df2a-93e0-4d93-ac70-c260a4628a32.JPG)

Hello.

On hiveos i can get 81 mh/s eth and 2.3 gh/s ton with core locked to 1410 and power limited to 340w; but with same computer and same settings i can only get 50mh and 3gh/s while dual mining which i think caused by power limitation. Even i set power limit to %100 lolminer only use 268watt at max. I dont know what im doing wrong, i need your help. 

Thank you.

**Comment by Kostas-13** [comment]
At pc u say 3gh Ton compared to 2.3 so it seems most of the power goes there and bottlenecks mem regarding volts. mb try lowering clocks, or and lock Vcore , or and use nvidiaProfileInspector 
@5 settings window:
Cuda - Force P2 State => set it OFF
Power management mode => prefer maximum performance

I'm on 3080s non lhrs got 725mV locked // 1285 effective clock // 10865mem (using nicehash quickminer for OC settings but without mining there too ofc) => @268w 102 eth // 1850 ton

Hope it helps. xD
